### PR TITLE
SASS-1845 Updated save and continue for benefits and updated method t…

### PR DESCRIPTION
**Pull request by BenCosfordHMRC**
…o work for SL

https://jira.tools.tax.service.gov.uk/browse/SASS-1845

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [x]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [x]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [x]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [x]  Have you checked the PR Builder passes?

### CODE REVIEW

**Comment by matt712** [comment]
uk.gov.hmrc: bootstrap-frontend-play-28 and  uk.gov.hmrc: bootstrap-test-play-28 could be bumped to 5.20.0

File: it/controllers/employment/CheckYourBenefitsControllerISpec.scala [hunk 1]
```diff
@@ -28,7 +28,7 @@ import builders.models.mongo.EmploymentUserDataBuilder.anEmploymentUserData
 import helpers.SessionCookieCrumbler.getSessionMap
 import models.benefits.{AccommodationRelocationModel, Benefits, BenefitsViewModel}
 import models.employment.createUpdate.{CreateUpdateEmployment, CreateUpdateEmploymentData, CreateUpdateEmploymentRequest, CreateUpdatePay}
```
Comment (ina-tu) [inline it/controllers/employment/CheckYourBenefitsControllerISpec.scala:30]: unused imports

File: test/services/EmploymentSessionServiceSpec.scala [hunk 2]
```diff
@@ -137,7 +139,16 @@ class EmploymentSessionServiceSpec extends UnitTest
           assetsModel = Some(AssetsModel(Some(true), Some(true), Some(100.00), Some(true), Some(100.00))),
           submittedOn = Some("2020-02-04T05:01:01Z"), isUsingCustomerData = true
         )
-      ))
+      ),
+      studentLoans = Some(
+        StudentLoansCYAModel(
+          true,
```
Comment (ina-tu) [inline test/services/EmploymentSessionServiceSpec.scala:145]: name boolean parameters warnings

**Review by ina-tu — CHANGES_REQUESTED** [review]
- coverage is a bit low for StudentLoansCYAModel
- needs a rebase

**Review by ina-tu — APPROVED** [review]

**Pull request merged**